packages/host/src/AppDrawer.tsx

```diff
@@ -62,21 +96,63 @@ const ListItemLink: React.FC<ListItemLinkProps> = props => {
       )),
     [props.to]
   );
+  const className = clsx(
+    props.classes['drawerMenuItem'],
+    !!selected && props.classes['drawerMenuItemSelected']
+  );
 
   return (
     <li>
```

Comment:
this `li` seems unneeded?

Reply:
I took them out and bathed in the delight of minimalism, and then noticed that it was all wrapped in a `List` so added them back in to be syntactically complete 🤷‍♂️

Reply:
Ah I just read the docs - how silly that `*L*ist*I*tem` isn't rendering a `<*l**i*>` 😢